js/template-card.js

```diff
@@ -0,0 +1,136 @@
+import {createOffers} from './util.js';
+
+const templateFragment = document.querySelector('#card').content;
+const template = templateFragment.querySelector('.popup');
+const fragment = document.createDocumentFragment();
+const advertElement = template.cloneNode(true);
+const childElements = advertElement.querySelectorAll('*');
+for (const childNode of childElements) {
+  childNode.classList.add('hidden');
+}
+const offerTypeToReadable = {
+  'flat': 'Квартира',
+  'bungalow': 'Бунгало',
+  'house': 'Дом',
+  'palace': 'Дворец',
+  'hotel': 'Отель',
+};
+const similarOffers  = createOffers(1);
+const mapCanvas = document.querySelector('#map-canvas');
+const OFFER_FOR_POPUP_IDX = 0;
+const offerForPopup = similarOffers[OFFER_FOR_POPUP_IDX];
+const author = offerForPopup.author;
+const offer = offerForPopup.offer;
```

Comment:
```suggestion
const { author, offer } = offerForPopup;
```
И это должно быть в функции рендера попапа